atom/browser/api/atom_api_window.cc

```diff
@@ -52,6 +52,11 @@ namespace api {
 
 namespace {
 
+// The DeprecatedOptionsCheckCallback funtion which is implemented in JavaScript
```

Comment:
`funtion` -> `function`
Even better: "This callback function is implemented in JavaScript"